internal/pkg/service/common/etcdop/iterator/iterator.go

```diff
@@ -37,6 +37,12 @@ type Iterator struct {
 	currentValue *op.KeyValue   // currentValue in the page, match currentIndex
 }
 
+// ForEachOp definition, it can be part of a transaction.
+type ForEachOp struct {
+	def Definition
+	fn  func(value *op.KeyValue, header *Header) error
+}
+
```

Comment:
Tu je klucova cast, ... pridane `ForEachOp/ForEachOpT`:
- teda da sa vytvorit definicia `GetAll().ForEach` a pouzit/spusti na inom mieste
- implementuje to `Op` interface, takze sa to da pridat ako sucast transakcie
- je to zlozene z `def` definicie iteratora a `fn` forEach funkcie.